source/features/index.js

```diff
@@ -0,0 +1,181 @@
+import groupBy from 'lodash.groupby';
+import sortBy from 'lodash.sortby';
+
+export const features = {
+	/* GENERAL */
+	keyboardShortcuts: {
+		id: 'feature-keyboard-shortcuts',
+		category: 'general',
+		label: 'Enable keyboard shortcut to toggle Night Mode (Alt + M)',
+		fn: require('./keyboard-shortcuts').default
+	},
+	preserveTextMessages: {
+		id: 'feature-preserve-text-messages',
+		category: 'general',
+		label: 'Preserve unsent text in the Messages modal when it closes',
+		fn: require('./preserve-text-messages').default
+	},
+	useSystemFont: {
+		id: 'feature-use-system-font',
+		category: 'general',
+		label: 'Use the system font',
+		init: true
+	},
+
+	/* HEADER */
+	addLikesButtonNavBar: {
+		id: 'feature-likes-button-navbar',
+		category: 'header',
+		label: 'Add a "Likes" button to the main navbar',
+		fn: require('./likes-button-navbar').default,
+		init: true
+	},
+	cleanNavbarDropdown: {
+		id: 'feature-clean-navbar-dropdown',
+		category: 'header',
+		label: 'Cleanup navbar dropdown',
+		fn: require('./clean-navbar-dropdown').default
+	},
+	cleanupSearchSuggestions: {
+		id: 'feature-cleanup-search-suggestions',
+		category: 'header',
+		label: 'Remove hashtags suggestions in the search popover',
+		init: true
+	},
+	hideMomentsTab: {
+		id: 'feature-hide-moments-tab',
+		category: 'header',
+		label: 'Hide "Moments" tab',
+		init: true
+	},
+	hideTwitterLogo: {
+		id: 'feature-hide-twitter-logo',
+		category: 'header',
+		label: 'Hide Twitter bird logo',
```

Comment:
I don't see any reason to disable this.

Reply:
Personal preference? I've disabled for me

Reply:
It has no value. You know you're on Twitter. We also need to space for the centering of the navbar tabs.

Reply:
It has value for me. I like to keep things the way they are. Also I click it sometimes to refresh the homepage.

Reply:
> We also need the space for centering of navbar tabs.

Reply:
> We also need the space for centering of navbar tabs

This was handled already; it only center the navbar if the bird logo is being hidden.

Reply:
What can I say, I'm just not interested in having this configurable.